tox.ini

```diff
@@ -2,4 +2,4 @@
 # Ignore node_modules and cloned repos when not in a virtual environment
 exclude = node_modules/*,tests/*,src/*
 max-line-length = 160
-ignore = C815,C816,W503,E203
+ignore = C815,C816,W503,E203,DTZ007,DTZ005,DTZ004
```

Comment:
Is there any reason to disable the these rules? The changes required when they're enabled look minimal.

Reply:
I left it out but can fix them, changes are minimal as you said but didn't want to jump the gun, so I waited for reviewer's opinion first.

Reply:
That fair enough. I think we should fix them, as they relate to similar recommendations in the Python docs.

Reply:
Ok, added all recommendations now, probably they can still be refactored slightly. Thanks!